frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/SaveExport/index.tsx

```diff
@@ -734,7 +734,7 @@ function download_({
         tissues_expanded: expandedTissues,
         tissue_filter: selectedFilters.tissues,
         genes: selectedGenes,
-        cell_types_selected: filteredCellTypes,
+        cell_types_selected: [...filteredCellTypes],
```

Comment:
why do we need to create a new array here? Thanks!

Reply:
transforming a set to list

Reply:
ah cool! Thank you!